basis/io/directories/windows/windows.factor

```diff
@@ -36,10 +36,6 @@ M: windows delete-file ( path -- )
     [ (delete-file) ]
     [ \ file-delete-failed boa rethrow ] recover ;
 
-M: windows copy-file ( from to -- )
-    make-parent-directories
-    [ normalize-path ] bi@ 0 CopyFile win32-error=0/f ;
```

Comment:
Is our new ``copy-file`` faster than the old one that used ``CopyFile``?  What is the main reason we want to not call ``CopyFile``?